chapters/hub/hub-api.md

```diff
@@ -0,0 +1,67 @@
+# Hub API
+
+Jina Hub exposes several handy commands, namely `list`, `build` and `push` with configurable parameters. These APIs constitute the user’s experience using `jina cli` for a seamless experience with the Hub.
+
+
+## `jina hub` installation:
+`hub cli` can be installed using pip, run `pip install "jina[hub]"`
+
+## `jina hub list`:
+`hub list` interface helps the user to retrieve information related to executor images in the Hub.
+It accepts filters like:
+ `name` corresponding to image name
+ `kind` corresponding to the kind of hub image (indexer / encoder / segmenter / crafter / evaluator / ranker etc)
+ `type` indicating either pod or app
+ `keywords` for filtering on specific terms like `sklearn`,  etc.
+
+`list` could be used for listing both local as well as remote images.
+
+
+## `jina hub new`:
+Jina provides an easy to use interface for spinning up an executor or an app
+### Create a new executor
+`jina hub new --type pod`
+It will start a wizard in CLI to guide you create your first executor. The resulted file structure should look like the following:
```

Comment:
Executor

Reply:
`resulted` -> `resulting`

Reply:
It will start a wizard in **the** CLI to guide you **in creating** your first executor.